cmd/pipeline/deploy.go

```diff
@@ -116,15 +121,22 @@ func deploy(ctx context.Context) *cobra.Command {
 			if err := deployPipeline(ctx, name, namespace, repository, branch, filename, wait, timeout, variables); err != nil {
 				return err
 			}
+
+			pipelineURL, err := getPipelineURL(ctx, name, namespace)
+			if err != nil {
+				return err
+			}
 			if !wait {
 				log.Success("Pipeline '%s' scheduled for deployment", name)
+				log.Information("Follow your pipeline progress via the URL:\n    %s", pipelineURL)
```

Comment:
@rberrelleza this is a deploy success without `--wait`, which sentence do you propose?

Reply:
> @jLopezbarb @rberrelleza It looks to me like we should definitely leave de URL in a new line, it's easier to see and there will be more space since most URLs will be quite long.
> 
> I would suggest to separate it into two type of outputs:
> 
> 1. A `success` (in green) line.
> 2. An `info` (in blue) line telling the developer that the pipeline is accessible via the following URL.
> 
> ```
> ✓  Pipeline 'movies' scheduled for deployment.
> i  Follow your pipeline progress via the URL:
>    https://staging.okteto.dev/#/spaces/jlopezbarb?resourceId=xxxx-xxx-xxx-xxxxx
> ```
> 
> What do you think?
> 
> We could also add a bit of space between outputs:
> 
> ```
> ✓  Pipeline 'movies' scheduled for deployment.
> 
> i  Follow your pipeline progress via the URL:
>    https://staging.okteto.dev/#/spaces/jlopezbarb?resourceId=xxxx-xxx-xxx-xxxxx
> ```
> 
> Not so sure about the text though 😝

@melissa-okteto what do you think about the displayed messages?  The output would need to handle three potential  states:
1. The pipeline was deployed successfully
2. The pipeline started, but it is still running
3. The pipeline failed to execute. 

In all three cases we also want to include a URL back to Okteto